common/darwin/Classes/FlutterRTCMediaStream.m

```diff
@@ -321,6 +321,58 @@ - (int)getConstrainInt:(NSDictionary*)constraints forKey:(NSString*)key {
   return 0;
 }
 
+- (RTCMediaStreamTrack*)cloneTrack:(nonnull NSString*)trackId {
+    NSString* newTrackId = [[NSUUID UUID] UUIDString];
+
+    RTCMediaStreamTrack *originalTrack = [self trackForId:trackId peerConnectionId: nil];
+    LocalVideoTrack* originalLocalTrack = self.localTracks[trackId];
+
+    if ([originalTrack.kind isEqualToString:@"audio"]) {
+      RTCVideoTrack *originalAudioTrack = (RTCAudioTrack *)originalTrack;
+
+      RTCAudioTrack* audioTrack = [self.peerConnectionFactory audioTrackWithTrackId:trackId];
```

Comment:
What about the audioSource? On native side, we fetch an audioSource that we use to initialise the audioTrack (similar to how video tracks are being init)?